internal/cmd/util_unix.go

```diff
@@ -17,6 +21,64 @@ func fileInfoUID(info fs.FileInfo) int {
 	return int(info.Sys().(*syscall.Stat_t).Uid) //nolint:forcetypeassert
 }
 
+func darwinVersion() (map[string]any, error) {
+	if runtime.GOOS != "darwin" {
+		return nil, nil
+	}
+	darwinVersion := make(map[string]any)
+	cmdOutput, err := exec.Command("sw_vers").Output()
+	if err != nil {
+		return nil, err
+	}
+	output := string(cmdOutput)
+	lines := strings.Split(output, "\n")
+	for _, line := range lines {
+		if len(strings.TrimSpace(line)) == 0 {
+			continue
+		}
+		parts := strings.Split(line, ":")
+		if len(parts) != 2 {
+			continue
+		}
+		key := strings.TrimSpace(parts[0])
+		key = strings.ToLower(key[:1]) + key[1:]
+		value := strings.TrimSpace(parts[1])
+		darwinVersion[key] = value
```

Comment:
This worries me a little. If there is any localization of the output (it seems unlikely because it *looks* like it's programmatic output), this and the next step will not produce the documented result.

It looks like it will work, but I’m not sure which of the three ways we have are best:

1. automatic translation as implemented
2. string matching (a case statement on the trimmed `key`)
3. multiple explicit calls as is being done for `--productVersionExtra`

Reply:
Let's do multiple calls. Do you want to have `--productName` there? It's unlikely chezmoi will run on something other than macOS.

Reply:
Let's keep it since it's part of the default output and if they change the output for any reason it could be interesting to know. (I would have hoped that `productName` would have been something like `macOS Sierra`, but no.)